### PR TITLE
Issue/bluetooth off behaviour

### LINKED ISSUE #1073: "Bluetooth is off" behaviour is not displaying expected results.
# Summary | Résumé

If Bluetooth is turned off, the user will be shown a screen that indicates Bluetooth is off. This is good if the user is in "monitoring" state, but if they are in "exposed" or "diagnosed" state, then this screen will prevent them from taking any further action.

While Bluetooth is required to collect and broadcast IDs, it is not required to submit daily keys or to see the current exposed state.
Screenshot_20200821-151050

# Steps to reproduce | Étapes à reproduire

### "Diagnosed" State
1. Launch the app.
2. Enter a one-time-key, and put the user in "diagnosed" state. Follow the prompt to upload the daily keys.
3. Return the app the next day.
4. Turn Bluetooth off. Launch the app.
5. The "Bluetooth Off" screen should appear, preventing the user from uploading the new daily keys.

### "Exposed" State
1. Launch the app.
2. Follow normal process to put this phone into an "exposed" state.
3. Turn Bluetooth off. Launch the app.
4. The "Bluetooth Off" screen should appear, preventing the user from seeing the screen telling them they are in an exposed state, along with the instructions on what to do.

# Environment | Environnement logiciel

iOS and Android

# Suggested Solution
- For the "exposed" and "diagnosed" states, display the normal content, but provide the error status at the bottom of the screen. See screenshots below for examples.
- Add an additional line of text to the error, below "COVID Alert is off", to indicate the specific problem being: "Bluetooth is off"

Exposure-ON-bottom
DiagnosedShare

### PR DESCRIPTION
**Pull request by jeberhardt**
# Summary | Résumé

Trello: https://trello.com/c/gGBbwjW2/321-%F0%9F%8E%A8-assess-bluetooth-is-off-behaviour-against-expected-results
Github: https://github.com/cds-snc/covid-alert-app/issues/1073

Adjusting the displayed content as described in Issue #1073. 

Closes #1073 

# Test instructions | Instructions pour tester la modification

1. Launch App
2. Enter One Time Key, but **DO NOT UPLOAD KEYS**, Hit Cancel after entering One Time Key.
3. Send app to background.
4. Turn off Bluetooth.
5. Return to app.
6. The app should prompt the user to upload their keys, but still at the bottom indicate that there is a problem with COVID Alert.

7. Similar steps to enter "Exposed" state, turn Bluetooth off, and return to app.

8. In "Monitoring" state, when Bluetooth is off, the error should be the content on the main part of the screen.

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

As discussed with @amazingphilippe, there is some design debt that remains from this issue

I believe the messaging for when the networking is off, and the user is in "Diagnosed" state should reflect that the action is to upload keys, not to perform exposure checks.

### CODE REVIEW

**Comment by timarney** [comment]
@emilykuret ☝️

**Comment by jeberhardt** [comment]
[🎨 Assess "Bluetooth is off" behaviour against expected results](https://trello.com/c/gGBbwjW2/321-%F0%9F%8E%A8-assess-bluetooth-is-off-behaviour-against-expected-results)

**Comment by emilykuret** [comment]
[Explore restructuring the menu](https://trello.com/c/wlYxBsAd/293-explore-restructuring-the-menu)

File: src/screens/home/HomeScreen.tsx [hunk 1]
```diff
@@ -89,33 +89,35 @@ const Content = ({setBackgroundColor, isBottomSheetExpanded}: ContentProps) => {
   switch (systemStatus) {
     case SystemStatus.Undefined:
       return null;
-    case SystemStatus.BluetoothOff:
-      return <BluetoothDisabledView />;
     case SystemStatus.Disabled:
```
Comment (amazingphilippe) [inline src/screens/home/HomeScreen.tsx:92]: Would these case also create the same issue of hiding code uploading instructions to the user? Or do these system status also prevent code uploading?

Comment (jeberhardt) [reply src/screens/home/HomeScreen.tsx:92]: This case is only for when the user is in the "monitoring" state, and would not apply when the user needs to upload codes.

**Comment by emilykuret** [comment]
[Change home screen to fix "Bluetooth is off" issue.](https://trello.com/c/uvgtCNQx/432-change-home-screen-to-fix-bluetooth-is-off-issue)

**Review by smcmurtry — APPROVED** [review]
Tested on Android and iOS - looks good to me.

**Pull request merged**